packages/components/src/chart/d3chart/style.scss

```diff
@@ -7,6 +7,7 @@
 .d3-chart__container {
 	position: relative;
 	width: 100%;
+	overflow-x: scroll;
```

Comment:
This is adding a scrollbar under all charts in Linux (probably Windows too).

![image](https://user-images.githubusercontent.com/3616980/51919923-b0f6f100-23e4-11e9-902c-a5ca424f78b2.png)

I took some time trying to figure out what was going on here, given that I [worked on this](https://github.com/woocommerce/wc-admin/pull/1226) some time ago. The issue seems to be that when the legend is on the left, `.d3-chart__container` doesn't adapt its size.

Adding:

```CSS
.woocommerce-chart__body-row .d3-chart__container {
	width: calc( 100% - 320px );
}
```

to `chart/d3chart/style.scss` seems to fix the issue for me.

Let me know if that helps. :+1:

Reply:
@Aljullu Thanks for digging into that and finding the answer! That fix is working for me as well. I've updated the PR.